pkgs/development/python-modules/asciitree/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "asciitree";
+  version = "0.3.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "4aa4b9b649f85e3fcb343363d97564aa1fb62e249677f2e18a96765145cc0f6e";
+  };
+
+  # tests not included with pypi release
+  doCheck = false;
```

Comment:
Tests are really important to make sure you're not getting broken by upstream packages or breaking downstream packages. Do you mind pulling from the 0.3.3 release on https://github.com/mbr/asciitree

Reply:
Great point tests have been added.